### DISCUSSION TITLE
Double click on desktop hide/show desktop icons

### DISCUSSION THREAD

**Discussion by Alexnoj** [ramensoftware/windhawk-mods#820] [Mod ideas]
Double click on desktop will hide/show desktop icons

**Comment by m417z** [comment]
Meanwhile, there's the [Click on empty taskbar space](https://windhawk.net/mods/taskbar-empty-space-clicks) mod that you can use for that.

**Reply by Alexnoj** [reply]
Know it. use it for other purpose.
Thanks!
